### ISSUE TITLE
Better a11y defaults in error handling docs

### ISSUE THREAD

**Issue by dreitzner** [ciscoheat/superforms-web#10]
Small improvements to the docs about error handling could have a great positive a11y impact on many future SvelteKit Superform sites.

Sections to improve:
- Validation Summary should have links to the formfields and should be focused
  - https://superforms.vercel.app/concepts/error-handling#listing-errors
  - https://www.w3.org/WAI/tutorials/forms/notifications/#listing-errors
- User notification of form field error (aria-described-by + aria-live)
  - https://superforms.vercel.app/concepts/error-handling#usage-client
  - https://www.w3.org/WAI/tutorials/forms/notifications/#on-focus-change

**Comment by ciscoheat** [comment]
Thank you, nice suggestions. I will also add to the [errorSelector option](https://superforms.vercel.app/concepts/error-handling#errorselector) `[aria-invalid]` for v1.0. Maybe `:invalid` as well?

**Comment by dreitzner** [comment]
Sounds like a good idea. Almost added it, but read somewhere that it will somehow be deprecated with some future WCAG spec, but might be wrong 😅.
[aria-invalid spec](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA21)

For the validation summary, there will be a need for a superforms update I think.
To list errors with links (e.g. `href="#email"`), there would be a need to have the id of the input available....
Or a way to generate an ID from the schema might be even better 😉

**Comment by ciscoheat** [comment]
Closing this, as https://formsnap.dev/ is the go-to for a11y with Superforms.
